src/System.Private.CoreLib/shared/System/BitOps.cs

```diff
@@ -11,14 +11,14 @@
 // Some routines inspired by the Stanford Bit Twiddling Hacks by Sean Eron Anderson:
 // http://graphics.stanford.edu/~seander/bithacks.html
 
-namespace System
+namespace System.Numerics
 {
     /// <summary>
     /// Utility methods for intrinsic bit-twiddling operations.
     /// The methods use hardware intrinsics when available on the underlying platform,
     /// otherwise they use optimized software fallbacks.
     /// </summary>
-    internal static class BitOps
+    public static class BitOperations
```

Comment:
nit: Not sure if anyone else cares; but I prefer to have the members ordered alphabetically; unless there is some good reason to order them otherwise.

It helps with discovering types and browsing source code when you aren't in an IDE.

Reply:
It also makes it easier to match this against the approved API and the reference assembly.

Reply:
Makes sense. Fixed